tests/x11/vlc.pm

```diff
@@ -43,6 +43,11 @@ sub run {
         wait_still_screen(1);
         assert_and_click 'close_vlc';
     }
+
+    if (!check_var('QEMUVGA', 'cirrus')) {
+        x11_start_program('vlc --no-autoscale --loop data/test.ogv', target_match => 'vlc-playing');
+        assert_and_click 'close_vlc';
```

Comment:
I'm not sure if `alt-f4` does work in every DM, then needle would be not needed.
Otherwise one needle can contain `close_vlc vlc-playing` tags with click area, as the needle is same.

Reply:
`close_vlc` is already there for the existing test.

Reply:
I guess so far, this wouldn't be a problem in the systems where VLC is being tested atm...

Reply:
Sure, but people will be very confused by many needles containing multiple tags.